### ISSUE TITLE
Variation-cognizant Probabilistic Power Flow Analysis via Multi-task Learning. (arXiv:2205.00598v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#18518]
Variation-cognizant Probabilistic Power Flow Analysis via Multi-task Learning. (arXiv:2205.00598v1 [eess.SY])<br>
https://ift.tt/1G3E6OT<br>
<p>With an increasing high penetration of solar photovoltaic generation in electric power grids, voltage phasors and branch power flows experience more severe fluctuations. In this context, probabilistic power flow (PPF) study aims at characterizing the statistical properties of the state of the system with respect to the random power injections. To avoid repeated power flow calculations involved in PPF study, the present paper leverages regression algorithms and neural networks to improve the estimation performance and speed up the computation. Specifically, based on the variation level of the voltage magnitude at each bus, we develop either a linear regression or a fully connected neural network to approximate the inverse AC power flow mappings. The proposed multi-task learning technique further improves the accuracy of branch flow estimation by incorporating the errors of voltage angle differences into the loss function design. Tested on IEEE-300 and IEEE-1354 bus systems with real data, the proposed methods achieve better performance in estimating voltage phasors and branch flows.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
